jerry-core/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set(JERRY_CORE_NAME jerry-core)
 project (${JERRY_CORE_NAME} C)
 
 # Optional features
-set(FEATURE_PROFILE         "full" CACHE STRING "Profile types: full, minimal")
+set(FEATURE_PROFILE         "es5.1-full" CACHE STRING "Profile types: es5.1-full, minimal, es2015-part")
```

Comment:
@jiangzidong I discussed this further with @LaszloLango and we came to the conclusion that using "es2015-subset" would be the best way to indicate that the profile is stable but incomplete. So [minimal, es5.1, es2015-subset] it is :)